tests/tasks/teardown/kit-cluster/teardown.yaml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
file name stutters here. I like the teardown.yaml that i had before. Wdyt ?

Reply:
Agree. Revised accordingly.